### ISSUE TITLE
Training adapters with EarlyStoppingCallback

### ISSUE THREAD

**Issue by sstojanoska** [Adapter-Hub/adapter-transformers#166]
Hi,

I have a question regarding the usage of EarlyStoppingCallback with Trainer.  
When training task adapter using your provided example script (AdapterHub_01_Adapter_Training.ipynb) with additional EarlyStoppingCallback I get unusual message ( when calling trainer.evaluate() or trainer.predict() )

```
There are adapters available but none are passed to model.forward
 ```
I tried to reproduce and it seems that it happens when there are available adapters but are not set to be active. This is not the case since ``` model.active_adapters ```  shows the adapter name and  ```model.has_adapters``` shows the model configuration together with the adapter. 

When I execute the script without early stopping callback there is no issue. Why this happens ?

**Comment by calpt** [comment]
Hi,

I'm not sure about the cause of this. Could you try to add
```
do_save_full_model=False,
do_save_adapters=True,
```
to the trainer initialization and see if that changes anything?

**Comment by calpt** [comment]
Hi, we've identified this as a bug in our modification of the `Trainer` class. It has been fixed in #178 and will be released with the next version.
